tests/network/functional/rollback_test.py

```diff
@@ -235,7 +235,7 @@ def test_setup_invalid_bridge_opts_fails(adapter, nic0):
     net_attrs = {
         'nic': nic0,
         'switch': 'legacy',
-        'custom': {'bridge_opts': 'foo=0'},
+        'custom': {'bridge_opts': 'foo=0 priority=1'},
```

Comment:
This tests failure so I wouldn't add the priority here.

Reply:
after rebase on the revert patches the two tests pass for me locally

Reply:
They do, but it does not make sense to test priority in this test if it fails on the foo option nevertheless